packages/docker-engine/docker.service

```
@@ -10,6 +10,7 @@ StartLimitIntervalSec=60s
 
 [Service]
 Type=notify
+EnvironmentFile=/etc/network/proxy/env
```

```suggestion
EnvironmentFile=/etc/network/proxy.env
```
I don't think we expect to have other proxy files, no need for more hierarchy.